Automattic/jetpack
# SSO: Ensure SSO Tooltips are accessible

Fixes https://github.com/Automattic/jetpack/issues/37300

## Proposed changes:

* This PR ensures that the SSO tooltip text that displays for 'SSO Status' and 'Send Invite'  on the Users > All Users admin page is accessible by making sure that the text displays on focus rather than on mouseover, and also uses the full text (SSO Status or Send Invite) to trigger that modal text, rather than the tooltip icon.
* Note: As the SSO feature has now also been added to the connection package but the module itself is still used within the Jetpack plugin, this PR focusses purely on addressing this specific SSO tooltip issue rather than doing that as well as making sure the Jetpack plugin uses only the package version of SSO files. That is being done here: https://github.com/Automattic/jetpack/pull/37153
* As such, the PHP changes have been made in both places (but the JS only in the package JS file which is referenced now in both places).

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

* To replicate the accessibility issue, on a test site with Jetpack connected, enable the SSO feature from the modules page at `/wp-admin/admin.php?page=jetpack_modules`.
* Navigate to Users > All Users.
* Add an additional user who does not have SSO enabled (do not 'invite user to WordPress.com' or you won't see the 'Sent invite' link and tooltip).
* Click on 'tab' until you are over 'SSO Status' or 'Send Invite' - neither will open up the tooltip text box.

To test the fix:
* Check out this PR locally or using the Beta tester plugin, and follow the same steps as above. 
* When focussing on 'SSO Status' you should see the relevant textbox, and the same with 'Send Invite' (including the tooltip icon and text)
* Note that this code is only testing the PHP changes from the module file (JS changes are directly in the package file version), but the same code is used in the package file as well.


### Screenshots:

Expected SSO Status tooltip display:
<img width="381" alt="Screenshot 2024-05-09 at 09 14 47" src="https://github.com/Automattic/jetpack/assets/16754605/4df9b575-27be-4bf6-bc04-0881f75d3750">


Expected Send Invite tooltip text display:

<img width="383" alt="Screenshot 2024-05-09 at 09 14 38" src="https://github.com/Automattic/jetpack/assets/16754605/064f3cb3-b39a-4494-89f2-033cde6e5431">
